src/pypit.py

```diff
@@ -10,7 +10,7 @@
 import traceback
 
 # Import PYPIT routines
-debug = True
+debug = []
```

Comment:
I'm all for this, but perhaps it's better to use a dictionary, instead of a list, where each dict key can take a True/False value. That way, one doesn't have to remember the appropriate key to put in the list in order to debug the arcs part of the code, for instance.

Either way, we should have a 'DEVELOP' key in the debug list/dict. This is a general debug that changes terminal printout for developers.